web/modules/custom/registration_form_custom/src/Services/ExtraFieldsHandler.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\registration_form_custom\Services;
+
+use Drupal\Core\Database\Connection;
+
+/**
+ * Class methods adds extra fields in custom DB.
+ */
+class ExtraFieldsHandler {
```

Comment:
FYI extra fields are functionality that exists in Drupal, so the name is misleading (it doesn't describe what this service does as well).
But you can leave it.